client/blocks/conversation-caterpillar/docs/fixtures.js

```diff
@@ -0,0 +1,30 @@
+export const comments = [
```

Comment:
idea: use an actual post/comment tree and pull it down on the fly like we do for reader subscription list items